# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

gpsrActionActionGoal action_goal
gpsrActionActionResult action_result
gpsrActionActionFeedback action_feedback
